XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -151,6 +151,7 @@
     <Compile Include="**\*.wpf.cs" />
     <Compile Include="**\*.wpf.*.cs" />
     <Compile Remove="**\Snackbar.shared.cs" />
+    <Compile Remove="**\AvatarView\AvatarView.shared.cs" />
```

Comment:
i think this line and a line above should not be there, right?

Reply:
fixed in SnackBarOptions PR